main.py

```diff
@@ -1,6 +1,17 @@
+import newrelic.agent
+import os
+
+#NEW_RELIC_LICENSE_KEY = Put license key here
+#ENVIRONMENT = Put environment here
+
+if os.environ.get('NEW_RELIC_LICENSE_KEY', None):
+    newrelic.agent.initialize(
+        config_file='newrelic.ini',
+        environment=os.environ.get('ENVIRONMENT', 'local')
+        )
+
 import argparse
 import inspect
-import os
 import yaml
 
 from logger import createLog
```

Comment:
Related to my other comment, I think if we can instrument Line 44 below (`runProcess()`) with a background task we can capture everything. The tricky part of this is that we only want that to happen when the process is NOT `APIProcess`. I think we can figure that out though and hopefully that would simplify our instrumentation process.